slack-api-client/src/main/java/com/slack/api/methods/impl/MethodsClientImpl.java

```diff
@@ -3221,6 +3245,126 @@ public StarsRemoveResponse starsRemove(RequestConfigurator<StarsRemoveRequest.St
         return starsRemove(req.configure(StarsRemoveRequest.builder()).build());
```

Comment:
🧮 nit(non-blocking): We can alphabetical order the list methods before these "stars" methods!